locales/en-US-x-pirate.json

```diff
@@ -2309,6 +2309,16 @@
         ]
       }
     },
+    "level_change": {
+      "phrases": {
+        "0": "Change to <LEVEL>."
+      },
+      "example_phrases": {
+        "0": [
+          "Change to Level 2."
+        ]
+      }
+    },
```

Comment:
WDYT about something more funny like "Hoist yer boots to <LEVEL>!" or "Make yer way to deck <LEVEL>!"? ;)

Reply:
hahah that second one is good, ship it! (pun intended)

Reply:
`<LEVEL>` can either be the level:ref (i.e. some textual representation of the level that differs from the numerical one ("Parking", "Lobby", etc.) or even numerical representations in countries where the ground floor is 1 instead of 0, see https://wiki.openstreetmap.org/wiki/Key:level:ref) or just the level information if there is no level:ref. In the latter case, `<LEVEL>` will be replaced with `level <number>`.